core/src/main/scala/org/apache/spark/scheduler/cluster/CoarseGrainedSchedulerBackend.scala

```diff
@@ -506,6 +522,21 @@ class CoarseGrainedSchedulerBackend(scheduler: TaskSchedulerImpl, val rpcEnv: Rp
       }
     }
 
+    conf.get(EXECUTOR_DECOMMISSION_CLEANUP_INTERVAL).map {
+      cleanup_interval =>
+      val cleanupTask = new Runnable() {
+        override def run(): Unit = Utils.tryLogNonFatalError {
+          val stragglers = executorsToDecommission.filter(executorsPendingDecommission.contains(_))
+          if (!stragglers.isEmpty) {
+            logError(
```

Comment:
Decommission is not guaranteed always by design and it's just our best effort approach. In addition, we delete it in any way at line 533. Shall we make Spark show `logWarn` instead of `logError`?

Reply:
Yeah good point, I think info might actually be better.